vm/src/obj/objdict.rs

```diff
@@ -244,6 +244,18 @@ fn dict_delitem(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     }
 }
 
+fn dict_clear(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(vm, args, required = [(dict, Some(vm.ctx.dict_type()))]);
+    let mut mut_obj = dict.borrow_mut();
+    match mut_obj.payload {
```

Comment:
Instead of matching on the payload, you can use the `get_mut_elements` helper, e.g.:

```rust
get_mut_elements(zelf).clear();
```

Reply:
Done